more doc changes (#97)
ISISNeutronMuon/quickBayes: docs/source/theory.rst

- The normalisation is just the value of the hyper volume for the parameters
- The :math:`P(D|\underline\theta, M)` can be represented as gaussians and approximated by a first order Taylor expansion

> In the second part I have extended the derivation to remove the first assumption, allowing for unique lines (functions) to be included.

Author: ```suggestion
In the second part the original derivation has been extended to remove the first assumption, allowing for unique lines (functions) to be included.
```
